astropy/units/core.py

```diff
@@ -1512,30 +1512,19 @@ def to_system(self, system):
         Returns
         -------
         units : list of `CompositeUnit`
-            The list is ranked so that units containing only the base
-            units of that system will appear first.
+            The list is that returned by `~astropy.units.UnitBase.compose`,
+            but sorted by the fewest number of non-base units, so that
+            preference is given to, e.g., "cm / s^2" over "Gal" (in cgs).
```

Comment:
This looks overly complicated and mentioning `compose()` seems to be needlessly exposing implementation details. It should be possible to simplify this description if the docstring had a few code examples.

Reply:
Good point. I added examples.